notebooks/MAST/HSC/CASJOBS/HCV_CASJOBS/environment.yml

```diff
@@ -0,0 +1,117 @@
+name: aptest
+channels:
+  - defaults
+  - conda-forge
+dependencies:
```

Comment:
can you make all these dependencies `>=` instead of `=`?  That will make it easier to manage going forward.

Reply:
sure

Reply:
done